source/unit/Fixture.ooc

```diff
@@ -122,7 +122,9 @@ Fixture: abstract class {
 	is ::= static IsConstraints new()
 	testsFailed: static Bool { get { This _testsFailed } }
 	printFailures: static func {
-		"Total time: %.2f s" printfln(This totalTime)
+		timeString := "Total time: %.2f s" format(This totalTime)
+		This _print(timeString)
```

Comment:
If `_print` would free its incoming argument we could get rid of a lot of `free()` calls to strings in this file.